webapp/src/components/Help.tsx

```diff
@@ -14,58 +15,223 @@ import { Stack, Container, useTheme } from '@mui/material'
   return (
-    <Container maxWidth={false} id="help" sx={{ textAlign: 'left', backgroundColor: theme.palette.background.default }}>
+    <Container
+      maxWidth={false}
+      id="help"
+      sx={{ textAlign: 'left', backgroundColor: theme.palette.background.default }}
+    >
       <Stack spacing={2}>
-        <h2>Welcome to the {EnvironmentVars.DOT_NAME} CV Manager Website</h2>
+        <h2>{`Welcome to the ${EnvironmentVars.DOT_NAME} CV Manager Website`}</h2>
         <p>
-          This application shows the physical location and message counts for each RSU installed by the Colorado
-          Department of Transportation at various road sites throughout Colorado.
+          This application helps organizations manage and monitor deployed RSUs, monitor and detect issues
+          with connected intersections, and view numerous data types all in one application.
         </p>
+
+        <h2>Organizations and Profiles</h2>
+        <p>
+          This application uses organizations to manage permissions. All devices and users within the CV
+          Manager are associated with one or more organizations. When using the CV Manager, you can only
+          view one organization at a time. At the top right corner, you will see the User Profile menu.
+          This menu allows the user to change organizations, if they are a member of multiple, as well as
+          logging out of the application.
+        </p>
+
+        <BorderedImage
+          src={organizationSelection}
+          alt="Profile Dropdown"
+          width="200"
+        />
+
+        <h2>Permissions</h2>
+
+        <p>
+          There are 3 roles within the CV Manager: User, Operator, and Admin. See the table below for the
+          permissions of each role (red = not allowed, green = allowed). Some users can also be granted
+          Super User permissions, which enable access to all resources within an organization and increase
+          their ability to move resources between organizations.
+        </p>
+
+        <BorderedImage
+          src={permissionsTable}
+          alt="Role Permissions Table"
+          width="400"
+        />
+
+        <h3>Note on feature flags</h3>
+
+        <ul>
+          <li>ENABLE_RSU_FEATURES: ENABLED
+            <ul>
+              <li>Affects: RSU viewer, RSU status monitoring, RSU configuration</li>
+            </ul>
+          </li>
+          <li>ENABLE_INTERSECTION_FEATURES: ENABLED
+            <ul>
+              <li>Affects: Intersection map, Intersection dashboard, Conflict Monitor events, assessments, and notifications</li>
+            </ul>
+          </li>
+          <li>ENABLE_WZDX_FEATURES: DISABLED
+            <ul>
+              <li>Affects: WZDx message viewer</li>
+            </ul>
+          </li>
+          <li>ENABLE_MOOVE_AI_FEATURES: DISABLED
+            <ul>
+              <li>Affects: Moove AI hard braking event viewer</li>
+            </ul>
+          </li>
+        </ul>
```

Comment:
Add dynamic env var references, simpler descriptions, and haas alert feature flag
```suggestion
        <h3>Enabled Features</h3>
        <p>
          The CV Manager can be deployed with different feature sets enabled or disabled. In this application, the
          following features are currently enabled or disabled as shown below:
        </p>
        <ul>
          <li>RSU Monitoring and Configuration: {EnvironmentVars.ENABLE_RSU_FEATURES ? 'ENABLED' : 'DISABLED'}</li>
          <li>Intersection Map/Dashboard: {EnvironmentVars.ENABLE_INTERSECTION_FEATURES ? 'ENABLED' : 'DISABLED'}</li>
          <li>WZDx Viewer: {EnvironmentVars.ENABLE_WZDX_FEATURES ? 'ENABLED' : 'DISABLED'}</li>
          <li>Moove AI Viewer: {EnvironmentVars.ENABLE_MOOVE_AI_FEATURES ? 'ENABLED' : 'DISABLED'}</li>
          <li>HAAS Alert Viewer: {EnvironmentVars.ENABLE_HAAS_FEATURES ? 'ENABLED' : 'DISABLED'}</li>
        </ul>
```

Reply:
I committed this version but used "can have different features" instead of using the word deployed to avoid some confusion and make it a bit more simple.